pkg/util/timeutil/timeutil.go

```diff
@@ -0,0 +1,7 @@
+package timeutil
+
+import "time"
+
+func ParseISO8601(dateTime string) (time.Time, error) {
+	return time.Parse(dateTime, "2006-01-02T15:04:05-0700")
```

Comment:
Two issues:
1. The arguments are in the wrong order
2. Atlas can specify the time in UTC format (this is always true for access lists), which will cause an error: `2021-02-08T19:42:36Z`

Reply:
yes, you are right - this is fixed in the next PR :)

Reply:
sorry haven't replied to the second point:
MMS has the following formatting for the date: `yyyy-MM-dd'T'HH:mm:ssX` `X` in java is quite flexible - it can be the timestamp of any format, starting from `Z` and ending with `+0800`.